src/containers/App/partials/AppHeader.js

```diff
@@ -53,16 +58,25 @@ const AppHeaderBrandLink = styled(RouterLink, {
   name: 'AppHeader',
   slot: 'BrandLink',
 })({
-  position: 'absolute',
-  left: '50%',
-  transform: 'translateX(-50%)',
   color: 'inherit',
   '& > svg': {
     display: 'block',
-    width: 'auto',
+    width: '110px',
+    height: 'auto',
   },
 })
 
+const AppHeaderNav = styled('nav', {
+  name: 'AppHeader',
+  slot: 'List',
+})(({ theme }) => ({
+  color: theme.palette.secondary.light,
+  display: 'flex',
+  flexDirection: 'row',
+  gap: theme.spacing(3),
+  margin: theme.spacing(0, 10),
+}))
```

Comment:
We can add a media query to hide this desktop nav links on small screens.

Add an opposite media query to the mobile nav (AppNavDrawer.js) that hides the mobile nav on large screens .

Reply:
The media query for displaying hamburger and nav links are not synced. We can use breakpoints from theme like this.
`[theme.breakpoints.up('sm')]: {
    // CSS here
  },`
  Look at ArticleSlideShow -block as reference.